src/writer/proposal.ts

```diff
@@ -57,11 +57,14 @@ export async function verify(body): Promise<any> {
   }
 
   const onlyAuthors = space.filters?.onlyMembers;
-  const authors = (space.members || []).map(address => address.toLowerCase());
-  const isAuthor = authors.includes(body.address.toLowerCase());
-  if (onlyAuthors && !isAuthor) return Promise.reject('only space authors can propose');
+  const members = (space.members || [])
+    .concat(...(space.admins || []), ...(space.moderators || []))
+    .map(member => member.toLowerCase());
```

Comment:
Why both concat and spread? any advantages?

I feel this is much easier to read:
```suggestion
  const members = (space.members || [])
    .concat(space.admins || [])
    .concat(space.moderators || [])
    .map(member => member.toLowerCase());
```

or maybe
```suggestion
  const members = [
    ...(space.members || []),
    ...(space.admins || []),
    ...(space.moderators || [])
  ].map(member => member.toLowerCase());
```

Reply:
Ops its a mistake I started with concat then changed to spread without removing concat